### ISSUE TITLE
Registration Error can't be resolved with reinstall

### ISSUE THREAD

**Issue by BuzzwordChief** [slynch8/10x#1292]
Hello,
since a few releases I get a registration error message ever other hour when I try to start 10x. No obvious pattern, multiple restarts don't resolve the issue just waiting a bit (1-3 hours). 
I reinstalled 10x twice + deleting the appdata/10x folder, just copied my settings file after reinstall.

![image](https://user-images.githubusercontent.com/11695209/206508506-484321b1-32f0-47f3-ade2-69b998cd22a2.png)

**Comment by slynch8** [comment]
also, if you haven't already try re-submitting your key

**Comment by slynch8** [comment]
Looks like something has gone wrong with the registry key.

Can you go here in your registry:
Computer\HKEY_LOCAL_MACHINE\SOFTWARE\PureDevSoftware\10x

and delete the `install_id` key

The re-install

**Comment by BuzzwordChief** [comment]
I deleted the key and reinstalled 10x. It worked for a few hours but now I get the same error again:

```
08/12/2022 23:04:30: puredevreg: machine_id success: (123)
08/12/2022 23:04:30: 
08/12/2022 23:04:30: puredevreg: machine_id: 123
08/12/2022 23:04:30: 
08/12/2022 23:04:30: puredevreg: Failed to decrypt existing install value in registry: A0pD7vQdeJ2DJ3IN8L9wyIS1AK1rzM7WZdjFK1BPARyXcI
08/12/2022 23:04:30: puredevreg: installed: 00/00/0000 00:00:00:0000 Sunday
08/12/2022 23:04:30: puredevreg: installed: false date: 00/00/0000 00:00:00:0000 Sunday
```

Launch after reinstall:
```
08/12/2022 20:48:49: puredevreg: machine_id: 123
08/12/2022 20:48:49: 
08/12/2022 20:48:49: puredevreg: installed: 02/08/2022 12:34:15:0742 Tuesday
08/12/2022 20:48:49: puredevreg: verifying install with database...
08/12/2022 20:48:49: puredevreg: installed: true date: 02/08/2022 12:34:15:0742 Tuesday
08/12/2022 20:48:49: puredevreg: registered: false
08/12/2022 20:48:49: Writing bat file: C:/Users/DaveM/AppData/Local/Temp/10xLogEnv.bat
08/12/2022 20:48:49: @echo off
08/12/2022 20:48:49: setlocal
08/12/2022 20:48:49: call "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\Tools\VsDevCmd.bat" > nul
08/12/2022 20:48:49: "C:/Program Files/PureDevSoftware/10x/10x.exe" -logenv "C:/Users/DaveM/AppData/Local/Temp/10xLogEnvOutput.txt"
08/12/2022 20:48:49: Executing command: cmd.exe /c C:/Users/DaveM/AppData/Local/Temp/10xLogEnv.bat
08/12/2022 20:48:49: Updating colours
08/12/2022 20:48:49: Updating UI from settings
08/12/2022 20:48:49: [10x] Opening workspace 
08/12/2022 20:48:49: Reading workspace settings: Default
08/12/2022 20:48:49: Reading backup files
08/12/2022 20:48:49: [10x] N10XMain::InitialiseApp time: 687.71 ms
08/12/2022 20:48:49: Initialise complete
08/12/2022 20:48:49: Python initialise finished
08/12/2022 20:48:49: Failed to find state file: C:/Users/DaveM/AppData/Roaming/10x/State/10xToolState.xml
08/12/2022 20:48:49: WindowsMain::Initialise time: 739.433594 ms
08/12/2022 20:48:49: Opening last workspace: C:/dev/ADC/adc
08/12/2022 20:48:49: [10x] Opening workspace C:/dev/ADC/adc
08/12/2022 20:48:49: [10x] [error] Workspace file does not exist: C:/dev/ADC/adc
08/12/2022 20:48:49: Telemetry: Initialising
08/12/2022 20:48:49: [python] Executing script Custom 0ms
08/12/2022 20:48:49: [python] Executing script Custom 0ms
08/12/2022 20:48:49: [python] Executing script Example 0ms
08/12/2022 20:48:49: [python] Executing script Example 0ms
08/12/2022 20:48:49: [python] Executing script Snippets 0ms
08/12/2022 20:48:49: [python] Executing script Snippets 0ms
08/12/2022 20:48:49: [python] Executing script Utils 0ms
08/12/2022 20:48:49: [python] Executing script Utils 0ms
08/12/2022 20:48:49: [python] Executing script Vim 0ms
08/12/2022 20:48:49: [python] Executing script Vim 0ms
08/12/2022 20:48:49: puredevreg: install verified
```

**Comment by slynch8** [comment]
It sounds like the machine_id is changing. Are you on a VM or anything like that?

I'd rather not share it here, but if you [email me](https://www.puredevsoftware.com/contact.php) I'll give you build with some extra logging

**Comment by BuzzwordChief** [comment]
I deleted all the registry keys, the appdata folder and reinstalled 10x. It seems to work fine now.

If it happens again I will email you, for now I consider the problem solved :)

**Comment by slynch8** [comment]
I've found the underlying cause of this. Fixed (0.35.75)
